src/PactNet.Abstractions/Matchers/Match.cs

```diff
@@ -168,5 +168,10 @@ public static IMatcher Include(string example)
         {
             return new IncludeMatcher(example);
         }
+
+        public static IMatcher ArrayContains(dynamic[] variations)
```

Comment:
issue: The public API should be documented

Reply:
Noted, will amend

Reply:
This one still needs documenting

Reply:
Done